pkg/logging/logfields/logfields.go

```diff
@@ -466,4 +475,31 @@ const (
 
 	// Mode describes an operations mode
 	Mode = "mode"
+
+	// PubKey is the public key
+	PubKey = "pubKey"
+
+	// NodeIPv4 is the node IPv4 addr
+	NodeIPv4 = "nodeIPv4"
+
+	// NodeIPv6 is the node IPv4 addr
+	NodeIPv6 = "nodeIPv6"
+
+	// PodCIDRv4 is the pod IPv4 allocation CIDR
+	PodCIDRv4 = "podCIDRv4"
+
+	// PodCIDRv6 is the pod IPv4 allocation CIDR
+	PodCIDRv6 = "podCIDRv6"
+
+	// WgIPv4 is the Wireguard IPv4 tunnel addr
+	WgIPv4 = "wgIPv4"
+
+	// WgIPv6 is the Wireguard IPv4 tunnel addr
+	WgIPv6 = "wgIPv6"
+
```

Comment:
I believe all these lines have gotten pulled in accidentally. They must not be backported.

Reply:
It was part of https://github.com/cilium/cilium/pull/15453, was that a mistake in the first place?

Reply:
The ones I highlighted are definitely _not_ part of #15453. The only log fields added via #15453 are `Resource`, `Rule`, `AttachedENIs` and `ExpectedENIs`.

Reply:
Ok, I fixed this now.